projects/rocprofiler-systems/tests/rocprof-sys-testing.cmake

```
@@ -705,6 +705,15 @@ function(ROCPROFILER_SYSTEMS_ADD_TEST)
                     $<TARGET_FILE_DIR:${TEST_TARGET}>/${TEST_NAME}.inst ${TEST_RUN_ARGS}
                 WORKING_DIRECTORY ${PROJECT_BINARY_DIR}
             )
+
+            add_test(
+                NAME ${TEST_NAME}-binary-rewrite-cleanup
+                COMMAND
+                    ${CMAKE_COMMAND} -E rm -rf
```

Avoid any unnecessary printouts
```suggestion
                    ${CMAKE_COMMAND} -E rm -rf > /dev/null
```